lib/classes/board.js

```diff
@@ -83,5 +83,17 @@ GridGame.classes.board = function () {
 
   };
 
+  this.place_random_rock = function() {
+    var x = Math.floor(Math.random() * GridGame.width);
+    var y = Math.floor(Math.random() * GridGame.height);
+    var tile = this.tile(x, y);
+    if (tile.city || tile.rock) {
+      this.place_random_rock();
+      return false;
```

Comment:
As a small "did you know?":
If you use `return` from an `if` statement, the `else` there isn't necessary.   You can have that clause exist outside of the `else` clause, like:

```javascript
if (tile.city || tile.rock) {
  this.place_random_rock();
  return false;
}
tile.rock = true;
```

It can help simplify things as you work, and is a good self-defense over having a function `return` two different types (e.g; in one case, it returns a `Number` but in another case it returns `false`.  

HTH!